### PR TITLE
addeded flex profile

### PR DESCRIPTION
**Pull request by jakob-gross**

### CODE REVIEW

File: css-flexbox/css-flexbox_profile/css/styles.css [hunk 1]
```diff
@@ -24,6 +24,11 @@ body {
   max-width: 1200px;
 }
 
+.profile {
```
Comment (emyjamalian) [inline css-flexbox/css-flexbox_profile/css/styles.css:27]: may I ask why have you defined this?

Comment (emyjamalian) [reply css-flexbox/css-flexbox_profile/css/styles.css:27]: ok I saw the section has this class 🙌🏽

Comment (emyjamalian) [reply css-flexbox/css-flexbox_profile/css/styles.css:27]: though I think the change on the `.profile__image` is missing to center the image

File: css-flexbox/css-flexbox_profile/css/styles.css [hunk 2]
```diff
@@ -34,11 +39,15 @@ body {
   border-radius: 50%;
   max-width: 50%;
   height: auto;
+  align-self: center;
```
Comment (CheckersRT) [inline css-flexbox/css-flexbox_profile/css/styles.css:42]: I think the picture should be centred rather than the title but using align-self is correct :)

**Pull request merged**